microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -568,8 +568,34 @@ def _load_drivers_for_dpdk(self) -> None:
         modprobe.load(rmda_drivers)
         modprobe.load(mellanox_drivers)
 
+    def check_dpdk_support(self) -> None:
+        # check requirements according to:
+        # https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk
+        node = self.node
+        supported = False
+        if isinstance(node.os, Debian):
+            if isinstance(node.os, Ubuntu):
+                supported = node.os.information.version >= "18.4.0"
+            else:
+                supported = node.os.information.version >= "10.0.0"
+        elif isinstance(node.os, Redhat):
+            supported = node.os.information.version >= "7.5.0"
+        elif isinstance(node.os, Suse):
+            supported = node.os.information.version >= "15.0.0"
+        else:
+            # this OS is not supported
+            raise SkippedException(
```

Comment:
Use the UnsupportedOsException, it returns more details.

Reply:
updated to throw unsupporteddistro on unknown OS's. 
Unsupported but known should still throw Skipped, this check is intended to replace all the spots we skip 16.04 debian 9 etc and other EOL  branched.